core/src/main/java/org/pentaho/platform/scheduler2/quartz/EmbeddedQuartzSystemListener.java

```diff
@@ -163,6 +163,11 @@ public boolean startup( final IPentahoSession session ) {
     return result;
   }
 
+  protected void initializeScheduler()
+    throws org.pentaho.platform.api.scheduler2.SchedulerException {
```

Comment:
You don't need to include the fully qualified class name for the exception throws `SchedulerException` is just fine.

Reply:
The class is using both quartz's SchedulerException, and our own implementation, hence the fully qualified class name